src/docs/build-and-earn/testnet-guides/prerequisites.md

```diff
@@ -0,0 +1,132 @@
+- Install [Docker Compose](https://docs.docker.com/compose/install/) 1.29.1 or above
+- Install [Git](https://github.com/git-guides/install-git)
+- Install a code editor such as Nano, Atom, etc.
+
+Missing any of the prerequisites above can lead to errors and being unable to run your node. It is recommended to have the latest version of Docker and Docker compose installed. If you already installed them, you can check the version by running the following commands in your terminal:
+```
+docker -v
+docker-compose -v
+```
+
+## Setup the environment before running a node
+
+After installing all prerequisites, you can start to set up the environment:
+1. Create a directory to use as your workspace:
+```
+mkdir ~/cudos
+cd ~/cudos
+```
+2. Make sure that you are in the correct directory (cudos directory in this example)
+3. Clone the correct branches from the [CudosNode](https://github.com/CudoVentures/cudos-node), [CudosBuilders](https://github.com/CudoVentures/cudos-builders), and [CudosGravityBridge](https://github.com/CudoVentures/cosmos-gravity-bridge) repositories:
+```
+git clone --depth 1 -b testnet https://github.com/CudoVentures/cudos-node.git CudosNode
+git clone --depth 1 -b master  https://github.com/CudoVentures/cudos-builders.git CudosBuilders
+git clone --depth 1 -b master https://github.com/CudoVentures/cosmos-gravity-bridge.git CudosGravityBridge
+```
+4. Change to the root of each local repository and check out the following branches:
+```
+cd CudosNode
+git checkout gravity/sdk-0.43
+
+cd..
+cd CudosBuilders
+git checkout sdk-0.43
+
+cd..
+cd CudosGravityBridge
+git checkout cosmos-sdk-0.43
+```
+5. Put all cloned repositories in the same directory
+6. Rename the folders accordingly to exactly _CudosNode_, _CudosBuilders_, and _CudosGravityBridge_
+7. Navigate to the _CudosBuilders_ directory
+8. Build the docker image of the binary by running the command:
+```
+cd ./docker/binary-builder&& sudo docker-compose --env-file ./binary-builder.arg -f ./binary-builder.yml -p cudos-binary-builder build
+```
+
+Note that you need to repeat the same steps for each node that you want to create.
+
+## Create an account and set up a Keplr wallet
+
+1. You can use the docker terminal to locate the containers' ID:
+```
+docker container ls
+```
+2. Copy the CONTAINER_ID and run the command:
+```
+docker exec -it <container_id> bash
+```
+3. Create an account by running the command (write the mnemonic phrase in a safe place):
+```
+cudos-noded keys add <myKeyName>
+```
+4. Follow the guide [Set up a Keplr wallet and link it to your account on the Cudos testnet network](/docs/build-and-earn/getting-started/creating-a-keplr-wallet.md)
+
+Note that the **cudos-noded** is a command-line interface. It is the same command-line interface used for deploying smart contracts. You can get the list of all commands by running:
+```
+--help
+```
+
+## Notes for installing docker for the first time
```

Comment:
depending on the os/flavour, there could also be some additional steps to follow - eg, first removing any "distro-specific" docker components first in case of a linux os
because of wise variety of options and combinations, i think it would be better to just point the users to the corresponding official vendor documentation, eg:
- https://docs.docker.com/get-docker/
- https://docs.docker.com/compose/install/